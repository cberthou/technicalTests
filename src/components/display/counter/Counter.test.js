import React from 'react';
import { shallow } from 'enzyme';
import Counter from './Counter';

describe('src/components/display/counter', () => {
    it('should display the account balance', () => {
        const wrapper = shallow(<Counter count="1234" />);

        expect(wrapper).toMatchSnapshot();
    });
});
