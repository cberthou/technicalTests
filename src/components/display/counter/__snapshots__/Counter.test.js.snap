// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/display/counter should display the account balance 1`] = `
<div>
  1234
  $
</div>
`;
